tox.ini

```diff
@@ -45,13 +45,55 @@ deps =
     twcurrent: Twisted
     twtrunk: https://github.com/twisted/twisted/archive/trunk.zip
 
+    {trial,coverage}: mock
+
+    coverage: coverage
+
+
+whitelist_externals =
```

Comment:
Removed in 7aac72dea5336363cac59f4193442561e56da974 and 862acc53d0684e5b06972424af16c23024976cdb